resqpy/well/_trajectory.py

```diff
@@ -58,7 +58,7 @@ def __init__(
             cell_kji0_list = None,
             wellspec_file = None,
             spline_mode = 'cube',
-            deviation_survey_file = None,  # TODO: this file actually contains trajectory data --> rename argument!
+            ascii_trajectory_file = None,
             survey_file_space_separated = False,
```

Comment:
The name of this argument should also be changed from survey_file... to ascii_file_space_separated or something like that. (also change the text in the docstring for this argument please).